.github/workflows/ci.yaml

```
@@ -220,7 +220,9 @@ jobs:
   # Lint checks
   prek:
     name: "Pre-commit Checks"
-    runs-on: ubuntu-latest
+    # Bumped to ubuntu-32gb: ubuntu-latest was hitting resource issues, using
+    # this overkill runner until we can root-cause and right-size.
+    runs-on: ubuntu-32gb
```

actionlint (1.7.11)

[error] label "ubuntu-32gb" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)